jobs/SprintTag/Jenkinsfile

```diff
@@ -16,27 +16,37 @@ node{
     env.stash_manifest_path = "${manifest_name}"
     
     def repo_dir = pwd()
+    def TESTS = "${env.TESTS}"
     // Create an instance of UnitTest/UnitTest.groovy
     def unit_test = load("jobs/UnitTest/UnitTest.groovy")
+    // Create an instance of FunctionTest/FunctionTest.groovy
+    def function_test = load("jobs/FunctionTest/FunctionTest.groovy")
+
     def retry_times = Integer.valueOf(env.Test_Time);
     int failed_times = 0
     for (int i=0; i < retry_times; i++) {
         try{
             stage("Unit Test${i}"){
                 // Call the function runTest to run unit test               
-                unit_test.runTest(env.stash_manifest_name, env.stash_manifest_path , repo_dir)
+                unit_test.runTest(env.stash_manifest_name, env.stash_manifest_path, repo_dir)
             }
             stage("Function Test${i}"){
-                load("jobs/function_test/function_test.groovy")
+                function_test.run(TESTS, env.stash_manifest_name, env.stash_manifest_path, repo_dir)
             }
         } catch(error){
             echo "Caught: ${error}"
             failed_times += 1
+        } finally{
+            unit_test.archiveArtifactsToTarget("UnitTest${i}")
```

Comment:
Can you paste a snapshot of what's the final view of the SprintTag build artifacts ?